### ISSUE TITLE
New conda version

### ISSUE THREAD

**Issue by hiraksarkar** [COMBINE-lab/minnow#5]
We need to put a `release` version in conda, which aims to mitigate following problems,
1.  Overall better usability 
2.  Interaction with splatter mode and alevin mode is not obvious 
3. For unexpected scenarios, we need better error messages with are descriptive rather than having undefined behavior or seg-faults.
4. Debug messages should get removed.  

Features to be added,
1. The support for variable CB Length, UMI Length and Read Length
2. Make TwoPaCo usable from inside minnow, removing the need for using external GFA
3. Fewer command line options.

**Comment by hiraksarkar** [comment]
@annajbott I would keep a track of this to push this version.

**Comment by annajbott** [comment]
Cheers I'll keep an eye out

**Comment by annajbott** [comment]
Any updates on this?

**Comment by Acribbs** [comment]
Hi @hiraksarkar.  Is there any update on when a new Conda release is planned?

**Comment by hiraksarkar** [comment]
Yes, we actually added a couple of new features and change the version completely. I am expecting to push it around second week of June.
